Controllers/ShipmentTypeController.cs

```diff
@@ -37,7 +40,7 @@ public async Task<ActionResult<IEnumerable<ShipmentType>>> Get()
         }
 
         [HttpGet("{id}")]
-        public async Task<ActionResult<ShipmentType>> GetByIdCustomer(int id)
+        public async Task<ActionResult<ShipmentType>> GetByIdUser(int id)
```

Comment:
eh in kenapa nama methodnya jadi `GetByIdUser` hwhw

Reply:
yang sebelumnya juga salah ternyata :D

jadi `GetById` aja cukup kayaknya

Reply:
okk saya push lg mas